Dockerfile

```diff
@@ -41,4 +41,6 @@ COPY --from=static /app/dist /app/dist/
 
 # Collect all of the static files into the static folder
 RUN DJANGO_ENV=production python manage.py collectstatic
-CMD exec gunicorn developerportal.wsgi:application --bind=0.0.0.0:8000 --reload --workers=3
```

Comment:
I checked with @limed and this line looks redundant - docker-compose also runs it, and k8s also has a version of it

Reply:
:+1: here the only time this is really only useful is if you are running this using the `docker run` command. Since we are using lots of `docker-compose` this is really not needed

Reply:
@limed Great, thanks. Just so I get it right in my head, is Prod using _anything_ defined in `docker-compose` at all?

Reply:
No nothing in k8s will be using `docker-compose`. See: https://github.com/mdn/developer-portal/blob/master/k8s/app.deploy.yaml.j2#L29